samples/LiveEffect/src/main/java/com/google/sample/oboe/liveEffect/LiveEffectEngine.java

```diff
@@ -39,4 +36,17 @@
     static native void setRecordingDeviceId(int deviceId);
     static native void setPlaybackDeviceId(int deviceId);
     static native void delete();
-}
+    static native void native_setDefaultStreamValues(int defaultSampleRate, int defaultFramesPerBurst);
+
+    static void setDefaultStreamValues(Context context) {
```

Comment:
Does this need @TargetApi(16) ?

Reply:
Since Oboe's minimum API level is 16 I don't think we need this.